src/librustc/dep_graph/graph.rs

```diff
@@ -644,15 +664,26 @@ pub(super) struct CurrentDepGraph {
     edges: IndexVec<DepNodeIndexNew, Vec<DepNodeIndexNew>>,
     node_to_node_index: FxHashMap<DepNode, DepNodeIndexNew>,
 
+    anon_id_seed: Fingerprint,
```

Comment:
this feels like it merits a nice comment

Reply:
Agreed `:)`